server/api/routes-auth.js

```diff
@@ -7,17 +7,29 @@ import koaRouter from 'koa-router'
 import svgCaptcha from 'svg-captcha'
 import consts from '../utils/consts'
 
+/**
+ * Feature flag whether or not we want to mock authentication.
+ * This should maybe in consts, or via .env file. TODO.
+ */
+const AUTH_BACKEND_ENDPOINT = '/foo/bar'
+const AUTH_BACKEND = consts.MOCK_AUTHENTICATION_ENDPOINT || AUTH_BACKEND_ENDPOINT
+
 const router = koaRouter({
   prefix: consts.BASE_API
 }) // router middleware for koa
 
+const headers = {
+  'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8'
+}
+
+if (typeof consts.MOCK_AUTHENTICATION_ENDPOINT === 'string') {
+  headers.Authorization = 'Basic YmFzLWNsaWVudDpYMmNYeW1nWkRrRkE3RWR0'
```

Comment:
This feels very specific to one environment, let's isolate this for only while we want that mocking auth server.